tests/Storage.test.ts

```diff
@@ -293,4 +294,111 @@ describe('LedgerStorage', () =>
                 });
         });
     });
+
+    it('Test the update preImage', (doneIt: () => void) =>
+    {
+        updatePreImageTest(doneIt);
+    });
 });
+
+/**
+ *  Run the test of the update preImage
+ */
+function updatePreImageTest (allDoneIt: () => void)
+{
+    let ledger_storage: LedgerStorage = new LedgerStorage(":memory:", (err1: Error | null) =>
+    {
+        assert.ok(!err1, err1?.message);
+
+        putAllBlockData(ledger_storage, (err2: Error | null) =>
+        {
+            assert.ok(!err2, err2?.message);
+
+            let height: number = 0;
+            ledger_storage.getEnrollments(height,
+                () =>
+                {
+                    it ('Update preImage normal Test', (doneIt: () => void) =>
```

Comment:
Note on naming: The function named `it` should be used as the first word of the sentence (but not put in the string). See [the example](https://mochajs.org/#getting-started).
So in this case, `"It " ~ "Update preImage normal Test"` is not quite a sentence.
Something like `it('tests that sending a pre-image with a distance of 6 works')` would be more appropriate.

Reply:
I improved the wording.